src/util/saveFiles.js

```diff
@@ -99,15 +99,17 @@ export const fetchAndSave = ({jsonQueue, mediaQueue}, cb) => {
 
   const fetchMedia = (err, medias, pagination, remaining) => {
     debugApi(`API calls left ${remaining}`)
-    debugApi(`Has next page ${!!pagination.next}`)
 
     if (err) {
+      console.log(err.error_type)
```

Comment:
It was only being used for debugging, debugApi(`API error ${err}`) will show that output